base-template.sh

```diff
@@ -703,10 +703,16 @@ show_prompt() {
         # else: Running fallback...
     fi
 
-    # Read from stdin
+    # Read from stdin if its available
     printf "%s %s [%s]:" "$TEXT" "$HINT_TEXT" "$SHORT_OPTIONS"
-    # shellcheck disable=SC2229
-    read -r "$VARIABLE" </dev/tty
+    if [ -f /dev/tty ]; then
```

Comment:
This should be done by the test setup, interactively we expect you have TTY elsewhere. Did you leave this change in by accident during testing?

Reply:
Please check this one, I'm not sure this change was needed.

Reply:
Out of curiosity: did you need this change for sure?
If not, I might revert, I think other places would have the same read with tty redirection.